install_config/aggregate_logging.adoc

```diff
@@ -12,31 +12,33 @@ toc::[]
 
 == Overview
 
-OpenShift cluster administrators can aggregate and view logs for a range of
-OpenShift services using the EFK stack, which is a modified version of the
-https://www.elastic.co/videos/introduction-to-the-elk-stack[ELK stack]. This
-topic shows you how to deploy the stack using a template.
+OpenShift cluster administrators can aggregate logs and
+enable users to view them for a range of OpenShift services
+using the EFK stack, which is a modified version of the
+https://www.elastic.co/videos/introduction-to-the-elk-stack[ELK
+stack]. The EFK stack can be useful for viewing logs aggregated from
+hosts and applications, whether coming from multiple containers or even
+deleted pods.
```

Comment:
Should it be s/users/application developers? I know the terms can be a bit confused in the docs, but we generally try to aim for the latter. OpenShift users/application users could be a confused term.

"Users" is used a few times in this PR.